classes/options-general.php

```diff
@@ -456,21 +456,21 @@ class='howto'><?php echo $api_how_to; ?></span>
 
 													<tr valign="top">
 														<td><label
-																for="units"><?php _e( 'Distance Units', 'simplemap' ); ?></label>
+																for="units"><?php _e( 'Distance Units', 'SimpleMap' ); ?></label>
 														</td>
 														<td>
 															<select name="units" id="units">
 																<option
-																	value="mi" <?php selected( $units, 'mi' ); ?>><?php _e( 'Miles', 'simplemap' ); ?></option>
+																	value="mi" <?php selected( $units, 'mi' ); ?>><?php _e( 'Miles', 'SimpleMap' ); ?></option>
 																<option
-																	value="km" <?php selected( $units, 'km' ); ?>><?php _e( 'Kilometers', 'simplemap' ); ?></option>
+																	value="km" <?php selected( $units, 'km' ); ?>><?php _e( 'Kilometers', 'SimpleMap' ); ?></option>
 															</select>
 														</td>
 													</tr>
 
 													<tr valign="top">
 														<td><label
-																for="default_radius"><?php _e( 'Default Search Radius', 'simplemap' ); ?></label>
+																for="default_radius"><?php _e( 'Default Search Radius', 'SimpleMap' ); ?></label>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 145 characters

<!-- Comment by [SideCI](https://sideci.com) -->